README.md

```diff
@@ -24,7 +24,7 @@ Before submitting a PR, make sure you run the linter & generate the go modules:
 protolint config.proto
 ```
 ```bash
-make generate-proto
+make generate-proto generate-env-vars
```

Comment:
why this is suddenly needed?  I thought it always generates the corresponding env var for the config field

Reply:
`make generate-proto` does that but does not update ENV.md with the new env vars.